### ISSUE TITLE
"pages" field missing for 2012 proceedings

### ISSUE THREAD

**Issue by jacksongoode** [NIME-conference/NIME-bibliography#55]
I've noticed that the 2012 proceedings do not include the "pages" field. Not sure if these could be added but I wanted to mention the issue.

**Comment by alexarje** [comment]
Yes, good catch, and it would be good to add the page numbers. If anyone has the full proceedings book available, it would be great to get these numbers added (I threw out my book at the Seoul airport due to overweight on my way back from the conference...).

**Comment by cpmpercussion** [comment]
I think there's a broader issue that some proceedings were generated without pages. I wonder if it would be ok (and not too revisionist) to reverse engineer page numbers and a paper number field from the proceedings.

I know that journal layout editors keep asking me for those page numbers so they are important for some things!

**Comment by alexarje** [comment]
Yes, I think it should be fine to create page numbers where they don't exist. Perhaps a script that can count the number of pages in the PDFs and generate sequences based on the paper order?
